zeeguu_api/api/dashboard.py

```diff
@@ -0,0 +1,288 @@
+from datetime import datetime
+
+import flask
+from flask import request
+
+from .utils.json_result import json_result
+from .utils.route_wrappers import cross_domain, with_session
+from . import api
+import zeeguu
+from zeeguu.model import User, Cohort, UserActivityData, Session, Language
+import sqlalchemy
+from flask import jsonify
+import json
+import random
+import datetime
+
+# class function wrapper
+def class_function_checker(class_id):
```

Comment:
comment is misleading function name does not help much.
- this is not a wrapper
- comments should say *why* a function is needed 
- function name it should say *what* it does ... in our case it seems to imply that it checks a class... checks for what?

Reply:
Made the comment more clear about function, no longer calling it a wrapper. New function name is 
'has_permission_for_class(class_id)'